templates/bundles/NelmioApiDocBundle/SwaggerUi/index.html.twig

```diff
@@ -24,7 +24,6 @@
         <div class="kbin-container">
             <nav class="head-nav"  data-action="click->kbin#toggleNav">
                 <div class="brand">
-                    <div><i class="fa-solid fa-bars"></i></div>
```

Comment:
this was a hamburger that appears on the `/api/docs` route only on mobile screen widths. clicking it did nothing. it seemed like there isn't a need for a hamburger on the api page so removed it